tests/cli/cases/dry-mode.js

```diff
@@ -27,6 +27,8 @@ shelljs.cd(TEST_DIRECTORY);
 
 const originalChangelog = shelljs.cat('CHANGELOG.md').stdout;
 
+console.log(`originalChangelog: ${originalChangelog}`);
+
```

Comment:
NIT: Is this supposed to be here?

Reply:
Great catch. I'll remove that log line and produce a callback version which is more idiomatic.

Reply:
Pushed just now.